# Provide Context and Understandability to Your Data

Metadata standards ensure that data are described using a consistent structure and format to provide necessary context for users across the data lifecycle from data management to accessibility and interoperability. In biological data, there are a few metadata standards to be aware of: the Ecological Metadata Language (EML), ISO-19115, and MIxS.

## Ecological Metadata Language (EML) üçÉ

### What Is It?

[Ecological Metadata Language](https://eml.ecoinformatics.org/eml-ecological-metadata-language) (EML) is a metadata schema (i.e. standard) that was developed by the ecological community for ecological data, including biological data. Shared data can include an EML file that provides context for all files in the data ‚Äúpackage‚Äù. EML is presented in [Extensible Markup Language](https://www.w3schools.com/xml/) (XML), which provides standard details for ecological data in a structure that is readable to both people and machines.

For more information, see the full [EML documentation](https://eml.ecoinformatics.org/eml-schema) or the National Center for Ecological Analysis and Synthesis (NCEAS)[EML GitHub repository](https://github.com/NCEAS/eml).

### Why?

-   It provides context and improves reproducibility of the data.

-   It captures important links and relationships between data, such as a time series, hierarchical taxonomies, IDs, and authoritative reference points and vocabularies.

-   EML helps represent ecological information in a standardized way.

-   EML is mandatory for [LTER](https://lternet.edu/), [iLTER](https://www.ilter.network/), [OBIS](https://obis.org/), [GBIF](https://www.gbif.org/), [Darwin Core Archive (DwC-A)](https://www.gbif.org/darwin-core) data sharing.

### **Top Resources**

Tools or packages to help write EML:

-   For data managers, coders:¬†

    -   [EML-R package](https://cran.r-project.org/web/packages/EML/index.html)

    -   [Postgresql database with fields compatible with EML](https://github.com/lter/LTER-core-metabase)

    -   [R-code for generating EML from LTER-metabase (built on EML-R package)](https://github.com/BLE-LTER/MetaEgress)

    -   [EMLAssemblyline (built on EML-R package)](https://ediorg.github.io/EMLassemblyline/articles/overview.html)

-   For those not inclined to write scripts:

    -   [ezEML](https://ezeml.edirepository.org/)

## ISO 19115 üó∫Ô∏è

### **What Is It?**

ISO-19115 is a metadata standard, developed and maintained by the [International Standards Organization](https://www.iso.org/home.html) (ISO), for describing geographic data. Biological data are inherently geographic, especially as we strive to understand how occurrences are impacted by ecological or environmental variables. ISO-19115 provides information about the identification, extent, quality, spatial and temporal schema, spatial reference, and distribution of geographic data. It evolved from the need for flexibility in harmonizing the Federal Geographic Data Committee (FGDC) Content Standard for Digital Geospatial Metadata (CSDGM) with other geospatial standards. For more information about implementations and extensions of ISO 19115, including for remotely sensed imagery and gridded data, see [the Digital Curation Centre ISO 19115 guide](https://www.dcc.ac.uk/resources/metadata-standards/iso-19115) or the [NCEI Metadata Workbook](https://www.ncei.noaa.gov/sites/default/files/2020-04/ISO%2019115-2%20Workbook_Part%20II%20Extentions%20for%20imagery%20and%20Gridded%20Data.pdf).

### Why?

-   It helps to provide important geographic context to data in a standardized way.

-   Using ISO metadata is mandatory for some U.S. federal agencies, like NOAA, NASA and USGS, to share their data through government repositories.

-   It can be used to describe individual files, data packages, and collections of datasets.

### Top Resources

-   [How to Convert ISO to EML](https://nceas.github.io/arcticdatautils/reference/convert_iso_to_eml.html)¬†

-   [Work Flow Model](https://www.fgdc.gov/metadata/iso-implementation-model-workflow)

-   [mdToolkit](https://www.mdtoolkit.org/home) - mdEditor is a writer for ISO 19115 metadata which uses mdJSON as an intermediary and mdTranslator allows translation to different metadata formats

## Minimum Information about any (x) Sequence (MIxS) üß¨

### What Is It?

[MIxS](https://w3id.org/mixs) (pronounced MIX-ess) is a set of checklists and packages for molecular genomic sequence data, such as DNA and RNA. MIxS is a standard published by the [Genome Standards Consortium (GSC)](https://www.gensc.org/) for molecular biologists and ecologists who create, manage, and archive sequence data. It includes a breadth of environment-specific metadata variables (e.g. soil variables) to augment genome-specific checklists (e.g. bacteria) and enables interoperability with environmental analyses.

### Why?

-   It helps to provide standardized metadata about genetic sequence data.

-   It is used by the International Nucleotide Sequence Database Collection ([INSDC](https://www.insdc.org/)), which has the following member participating databases: [ROIS - NIG](https://www.ddbj.nig.ac.jp/index-e.html), [EMBL-EBI](https://www.ebi.ac.uk/ena/browser/home) and [NCBI](https://submit.ncbi.nlm.nih.gov/).

### Top References

-   [MIxS Term Search Tool](http://www.gensc.org/pages/standards/search-terms.html)

-   [Genomic Standards Consortium term list](https://genomicsstandardsconsortium.github.io/mixs/)

-   [Minimum Information about Marker Gene Sequence (MIMARKS)](https://www.nature.com/articles/nbt.1823)

-   MIxS is maintained by the community using GitHub. To propose changes, ask questions, see the [MIxS GitHub repository](https://github.com/GenomicsStandardsConsortium/mixs).

-   [Minimum Information about Sequence Data from the Built Environment (MIxS-BE)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869023/)
