# Provide Context and Understandability to Your Data

Metadata standards ensure that data are described using a consistent structure and format to provide necessary context for users across the data lifecycle from data management to accessibility and interoperability. In biological data, there are a few important metadata standards to be aware of: the Ecological Metadata Language (EML), ISO-19115, and MIxS.

## Ecological Metadata Language (EML)

### What Is It?

[Ecological Metadata Language](https://eml.ecoinformatics.org/eml-ecological-metadata-language) (EML) is a metadata schema (i.e. standard) that was developed by the ecological community for ecological data, including biological data. Shared data can include an EML file that provides context for all files in the data “package”. EML is presented in [Extensible Markup Language (XML), which provides standard details for ecological data in a structure that is readable to both people and machines](https://www.w3schools.com/xml/).

For more information, see the full [EML documentation or the National Center for Ecological Analysis and Synthesis (NCEAS) ](https://eml.ecoinformatics.org/eml-schema)[EML GitHub repository](https://github.com/NCEAS/eml).

### Why?

-   It provides context and improves reproducibility of the data.

-   It captures important links and relationships between data, such as a time series, hierarchical taxonomies, IDs, and authoritative material.

-   EML helps represent ecological information in a standardized way.

-   EML is mandatory for [LTER](https://lternet.edu/), [iLTER](https://www.ilter.network/), [OBIS](https://obis.org/), [GBIF](https://www.gbif.org/), [Darwin Core Archive (DwC-A)](https://www.gbif.org/darwin-core) data sharing.

### **Top Resources**

Tools or packages to help write EML:

-   For data managers, coders: 

    -   [EML-R package](https://cran.r-project.org/web/packages/EML/index.html)

    -   [Postgresql database with fields compatible with EML](https://github.com/lter/LTER-core-metabase)

    -   [R-code for generating EML from LTER-metabase (built on EML-R package)](https://github.com/BLE-LTER/MetaEgress)

    -   [EMLAssemblyline (built on EML-R package)](https://ediorg.github.io/EMLassemblyline/articles/overview.html)

-   For scientists or those not inclined to write scripts:

    -   [ezEML](https://ezeml.edirepository.org/)

## ISO 19115

### What Is It?

Content standard for describing geographic data sponsored by the International Standards Organization (ISO). At its most basic, it is written in narrative form with class diagrams. There are many implementations and extensions (e.g., <https://www.dcc.ac.uk/resources/metadata-standards/iso-19115>).

### Why?

-   Provide context to your data (biological data is inherently 'geographic')

-   Standardized representation of information

-   **Mandated** by some US federal agencies, including NOAA, NASA, and USGS

-   Can be used at different granularities, used to describe data packages or collections, as well as at a dataset level (?): content standard vs collection standard?

### What?

-   Evolved from the need to harmonize the FGDC Content Standard for Digital Geospatial Metadata (CSDGM) with other formal and defacto standards that support the documentation of geospatial data and services.

-   Many variations including 19115, 19115-1, 19115-2. From [NCEI](https://www.ncei.noaa.gov/resources/metadata):

    -   ISO 19115 Geographic information -- Metadata: The ISO standard for documenting geospatial data. 

    -   ISO 19115-2 Geographic information -- Metadata -- Part 2: Extensions for imagery and gridded data: An extension of ISO 19115 used to document information about imagery, gridded data, and remotely sensed data. The root of ISO 19115 metadata records will change from MD_Metadata to MI_Metadata when using ISO 19115-2.

-   Usurped FGDC CSDGM - all users encouraged to migrate to ISO.

-   Highly flexible for many uses compared FGDC CSDGM, but few required elements leaves room for incomplete metadata

### Top References

-   [NOAA Workbook for ISO 19115-2](https://www.ncei.noaa.gov/sites/default/files/2020-04/ISO%2019115-2%20Workbook_Part%20II%20Extentions%20for%20imagery%20and%20Gridded%20Data.pdf) 

-   [How to Convert ISO to EML](https://nceas.github.io/arcticdatautils/reference/convert_iso_to_eml.html) 

-   [Work Flow Model](https://www.fgdc.gov/metadata/iso-implementation-model-workflow)

-   [mdToolkit](https://www.mdtoolkit.org/home) - mdEditor is a writer for ISO 19115 metadata which uses mdJSON as an intermediary and mdTranslator allows translation to different metadata formats

## Minimum Information about any (x) Sequence (MIxS)

### Who?

This is a standard for molecular data, like DNA and RNA. It is used by molecular biologists and ecologists who generate, manage and archive these type of sequence data.

### What Is It?

A set of checklists and packages for genomic sequence data.

### Why?

-   Provide minimal standardized metadata about genetic sequence data

-   Agreed upon and published by the Genome Standards Consortium

-   Used by the INSDC (DDBJ, EMBL-EBI and NCBI)

### Key Information

-   MIxS (pronounced MIX-ess), a suite of checklists standards, is introduced to report on a breadth of environment-specific metadata variables to augment the genome-specific checklists

-   Enables mixing and matching of genome checklists and environmental-specific packages

-   [![MIxS Structure](http://www.gensc.org/images/mixs_ext_graphic-1024x731.png)](http://www.gensc.org/pages/standards-intro.html)

### Top References

-   [MIxS Term Search Tool](http://www.gensc.org/pages/standards/search-terms.html)

-   [Genomic Standards Consortium term list](https://genomicsstandardsconsortium.github.io/mixs/)

-   [Minimum Information about Marker Gene Sequence (MIMARKS)](https://www.nature.com/articles/nbt.1823)

-   [MIxS GitHub repo](https://github.com/GenomicsStandardsConsortium/mixs)

-   [Minimum Information about Sequence Data from the Built Environment (MIxS-BE)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869023/)
